feature/home/src/main/java/com/moya/funch/navigation/HomeNavigation.kt

```diff
@@ -0,0 +1,35 @@
+package com.moya.funch.navigation
+
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.compose.composable
+import androidx.navigation.navigation
+import com.moya.funch.HomeRoute
+
+const val HOME_GRAPH_ROUTE = "home_graph"
+
+fun NavController.navigateToHome() =
+    navigate(Home.FunchHome.route) {
+        popUpTo(graph.id)
+    }
+
+fun NavGraphBuilder.homeGraph(
+    onNavigateToMyProfile: () -> Unit,
+    onNavigateToMatching: () -> Unit,
+) {
+    navigation(
+        route = HOME_GRAPH_ROUTE,
+        startDestination = Home.FunchHome.route,
+    ) {
+        composable(route = Home.FunchHome.route) {
+            HomeRoute(
+                onNavigateToMatching = onNavigateToMatching,
+                onNavigateToMyProfile = onNavigateToMyProfile,
+            )
+        }
+    }
+}
+
+internal sealed class Home(val route: String) {
+    data object FunchHome : Home("home")
+}
```

Comment:
```suggestion
internal sealed class HomeScreens(val route: String) {
    data object Home : Home("home")
}
```

`Home.FunchHome` 을 봤을 때 혼란을 줄 수 있다고 생각합니다.  
따라서, `HomeScreens.Home` 을 제안합니다~